src/layouts/default/PlayerSelect.vue

```diff
@@ -53,6 +53,11 @@
           "
           @click="playerClicked(player)"
         />
+        <PlayerCard
+          v-if="!webPlayer.player_id"
+          player="web"
+          @click="localPlayerEnabled"
+        />
```

Comment:
Some comment as before, I think the player should be treated as any other player and we just add some filter to only show it on its own device.